terminus/logutils.py

```diff
@@ -23,23 +24,24 @@ class RecordingAdapter(logging.LoggerAdapter):
     """
 
     def process(self, msg, kwargs):
-        extra = kwargs.get('extra', {})
-        if 'recording' in kwargs:
-            extra['device_id'] = kwargs['recording']['device_id']
-            extra['recording_id'] = kwargs['recording']['id']
+        extra = kwargs.get("extra", {})
+        if "recording" in kwargs:
+            recording = kwargs["recording"]
+            extra["device_id"] = str(uuid.UUID(bytes=recording.device_id))
```

Comment:
I would like to know why we do that, a comment is really appreciated

Reply:
The reason is in `messages/recording.proto`. The uuid are saved as array of bytes instead of strings so that we can save half the size on each message.